src/frontend/src/pages/HomePage/GuestHomePage.tsx

```diff
@@ -0,0 +1,39 @@
+import { Typography, Box } from '@mui/material';
+import PageLayout from '../../components/PageLayout';
+import { AuthenticatedUser } from 'shared';
+import ImageWithButton from './components/ImageWithButton';
+import React from 'react';
+
+interface GuestHomePageProps {
+  user: AuthenticatedUser;
+  setClickedFinishline: (e: boolean) => void;
+}
+
+const GuestHomePage = ({ user, setClickedFinishline }: GuestHomePageProps) => {
+  return (
+    <PageLayout title="Home" hidePageTitle>
+      <Typography variant="h3" textAlign="center" sx={{ mt: 2, pt: 3 }}>
+        {user ? `Welcome, ${user.firstName}!` : 'Welcome, Guest!'}
+      </Typography>
+      <Box sx={{ display: 'flex', mt: 4 }}>
+        <Box sx={{ display: 'flex', padding: '50px', gap: 5 }}>
+          <ImageWithButton
+            title="Interested in applying"
```

Comment:
I actually really like the way the text looks int the  mockup, could you match the text and try to  have the same thickness in font